open_humans/uwsgi.ini

```diff
@@ -0,0 +1,17 @@
+[uwsgi]
+die-on-term = true
+env = DJANGO_SETTINGS_MODULE=open_humans.settings
+gevent = 200
+harakiri = 120
+http-socket = :$(PORT)
+lazy = true
+master = true
+max-requests = 5000
+memory-report = false
+module = open_humans.wsgi:application
+processes = 4
+
+; If VIRTAL_ENV is set then use its value to specify the virtualenv directory
+if-env = VIRTUAL_ENV
```

Comment:
this is managed automatically by `virtualenv`; whenver you have a virtualenv activated (as you will after `workon <virtualenv-name`) it will be set :)